locales/en-GB/event-resources.properties

```diff
@@ -2,9 +2,9 @@ host_maker_party=Host A Maker Party
 host_maker_party_paragraph=Run your very own Maker Party event using these great tools and activities. Mobilise and educate your community to take action through making. We’ll provide the resources to plan, host and promote your event, providing support every step of the way.
 maker_party_activities=Maker Party Activities
 maker_party_activity_1_title=Meme School
-maker_party_activity_1_body=Create memes and discuss implications of outdated copyright law.
+maker_party_activity_1_body=Create memes that support copyright reform! You’ll learn how to use pictures from the public domain to gain a better understanding of the implications of outdated copyright law.
 maker_party_activity_2_title=Contributing to the commons
-maker_party_activity_2_body=Help people understand the value of content we can all use freely by contributing your own content under a Creative Commons licence.
+maker_party_activity_2_body=Help people understand the value of content we can all use freely by contributing your own content under a creative commons license.
```

Comment:
Yeah, again, someone went in and fixed en-GB, to add the caps, but en-US was still broken.

So I reverted the en-GB, but also, updated en-US to also be caps.